sw/device/silicon_creator/rom_ext/BUILD

```diff
@@ -92,6 +92,15 @@ ld_library(
     ],
 )
 
+ld_library(
+    name = "ld_va",
+    script = "rom_ext_va.ld",
```

Comment:
nit: "va" suffix is a bit cryptic. WDYT about `rom_ext_virtual_addr.ld`?

Reply:
done